Dockerfile

```diff
@@ -20,12 +20,21 @@ ENV        NODE_PATH=/calypso/server:/calypso/client
 COPY       ./env-config.sh /tmp/env-config.sh
 RUN        bash /tmp/env-config.sh
 
+# Build a node_modules layer
+#
+# This one builds out our node_modules tree. Since we use
+# file: references, we have to copy over our 
+# package.json, lockfiles, and the contents of packages/*
+COPY package.json /calypso/
+COPY npm-shrinkwrap.json /calypso/
```

Comment:
These can be combined into a single `COPY`:

```dockerfile
COPY package.json npm-shrinkwrap.json /calypso/
```

Reply:
To be clear, no need to make this change as it's nearly the same. There will be an extra layer created but for a single file copy it's pretty much irrelevant.

Reply:
done anyway :)